<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telegramBotDialog</class>
 <widget class="QDialog" name="telegramBotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenCV Face Recognition System - TelegramBot</string>
  </property>
  <widget class="QLabel" name="tipLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Please modify the following </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="tokenLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Please fill in the API Token of TelegramBot</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="telegramIDLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>Please fill in the recipient's Telegram ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="tokenLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>54</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>API Token：</string>
   </property>
  </widget>
  <widget class="QLabel" name="telegramIDLabel">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>104</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Who to send to:</string>
   </property>
  </widget>
  <widget class="QLabel" name="socksLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>154</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Socks Proxy:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Update configuration</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="socksLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Please fill in proxy_url, leave blank to use direct connection</string>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Content:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="messagePlainTextEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Please fill in the content of the message to be sent, only plain text is supported</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Created by KRC</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
